HB/about.elpi

```diff
@@ -61,23 +61,23 @@ main-located S (loc-abbreviation _) :- coq.error "HB: unknown abbreviation" S.
 
 shorten coq.pp.{ v , h, hv, hov , spc , str , box , glue , brk , empty }.
 
-pred bulletize1 i:(A -> coq.pp -> prop), i:A, o:coq.pp.
+func bulletize1 (func A -> coq.pp), A -> coq.pp.
 bulletize1 F X (glue [str "- ", M]) :- F X M.
 
-pred bulletize i:list A, i:(A -> coq.pp -> prop), o:coq.pp.
-bulletize [] _ empty.
+func bulletize list A, (func A -> coq.pp) -> coq.pp.
```

Comment:
Just to be sure, bulletize is a function?
If it is not the case, the cut in the rule below should be removed...